boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts

```diff
@@ -65,6 +65,6 @@ arduino_serial: &lpuart1 {};
 	current-speed = <115200>;
 };
 
-&usb1 {
+zephyr_usbd0: &usb1 {
```

Comment:
If this is for samples, we generally rely on aliases for this. Why was a node label chosen?

One potential problem with node labels is that they can't be overridden easily.

Reply:
The idea is to use it the same way as arduino_i2c or arduino_spi nodelabels, see next commit and discussion in the bug report.

Reply:
I'm with @mbolivar-nordic here on using aliases.  The `arduino_i2c` and `arduino_spi` labels have a somewhat hardware defined/based meaning.